app/models/exchange_rate_currency_rate.rb

```diff
@@ -14,10 +14,24 @@ class ExchangeRateCurrencyRate < Sequel::Model
            to: :exchange_rate_currency,
            allow_nil: true
 
+  def_column_accessor :country, :country_code
```

Comment:
For the unfamiliar, this essentially makes these fields that come from an association right join serializable in the api.